nmostesting/suites/IS1101Test.py

```diff
@@ -537,35 +537,31 @@ def test_02_02_04_02(self, test):
             )
             if response.status_code != 200:
                 return test.FAIL(
-                    "The request has not succeeded: {}".format(response.json())
+                    "Contraints active request for sender {} has failed: {}".format(sender_id, response.json())
                 )
             constraints = response.json()
 
             if not IS04Utils.compare_constraint_sets(
                 constraints["constraint_sets"],
                 self.sample_rate_constraints[sender_id]["constraint_sets"],
             ):
-                return test.FAIL(
-                    "constraints and SampleRateConstraints["
-                    + sender_id
-                    + "] are different "
-                )
+                return test.FAIL("Constraints and SampleRateConstraints[{}] are different".format(sender_id))
```

Comment:
Could be clearer?

Reply:
SampleRateConstraints doesn't mean anything to the test tool user. Could we try for a message the user can understand without reference to the code?